app/main.py

```diff
@@ -1,8 +1,39 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, list_of_cars: list) -> float:
```

Comment:
Be more specific in type annotation indicating a list of what entities you expect for the `list_of_cars` variable. Besides that, rename `list_of_cars` to `cars` there is no need to indicate type in a variable name here.

Reply:
fixed in [commit](https://github.com/job-artem/py-car-wash-station/commit/c7a74045fb18c03c88e7633921be4097e5f9e22c)